pkg/cmd/pr/status/status.go

```diff
@@ -57,6 +58,7 @@ func NewCmdStatus(f *cmdutil.Factory, runF func(*StatusOptions) error) *cobra.Co
 		},
 	}
 
+	cmd.Flags().BoolVar(&opts.ShowConflicts, "show-conflicts", false, "Show merge conflicts on pull requests")
```

Comment:
does it make sense for this to have a short alias, maybe `-c`?

Reply:
I think that makes sense to add the short alias since `--show-conflicts` is a bit verbose.
 
```suggestion
	cmd.Flags().BoolVar(&opts.ShowConflicts, "show-conflicts", false, "Display pull request merge conflict status")
```

I am not sure my wording is the best but I wanted the long description to clarify that this flag will show an indicator if the pull request has a merge conflict and not have confusion where people expect the whole merge conflict to be displayed.